languages.toml

```diff
@@ -2167,3 +2167,16 @@ comment-token = "#"
 [[grammar]]
 name = "hosts"
 source = { git = "https://github.com/ath3/tree-sitter-hosts", rev = "301b9379ce7dfc8bdbe2c2699a6887dcb73953f9" }
+
+[[language]]
+name = "fsharp"
+scope = "source.fsharp"
+injection-regex = "^fsharp$"
+file-types = ["fs"]
+roots = []
+comment-token = "//"
+indent = { tab-width = 4, unit = "    " }
```

Comment:
It would be good to add a configuration for https://github.com/fsharp/FsAutoComplete as well, it looks like that's the most popular language server for F#